apps/e2e/pages-storybook/src/app/shared/links/links.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component } from '@angular/core';
+import { SkyLinkListModule } from '@skyux/pages';
+
+@Component({
+  selector: 'app-links',
+  standalone: true,
+  imports: [SkyLinkListModule],
+  templateUrl: './links.component.html',
+  styles: `
+    :host {
+      display: contents;
```

Comment:
Would we expect our consumers to use `display: contents` if they wish to share links between multiple pages?

Reply:
It wasn't needed, I think I was checking something else. I've removed the extra styles.